Formula/opencsg.rb

```diff
@@ -0,0 +1,38 @@
+class Opencsg < Formula
+  desc "The CSG rendering library"
+  homepage "http://www.opencsg.org"
+  url "http://www.opencsg.org/OpenCSG-1.4.0.tar.gz"
+  sha256 "ecb46be54cfb8a338d2a9b62dec90ec8da6c769078c076f58147d4a6ba1c878d"
+
+  depends_on "qt5" => :build
+  depends_on "glew"
+
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/814a29d3ce4c6e7e919cd2fcd64bf45d421e821b/opencsg/patch-build.diff"
+    sha256 "9d710cf6c2d5495ca5ba51c0319785cefc21477c85fa3aacb9ccd3473fee54f3"
+  end
+
+  def install
+    system "qmake", "-r", "INSTALLDIR=#{prefix}",
+      "INCLUDEPATH+=#{HOMEBREW_PREFIX}/include",
```

Comment:
You don't want to include all of `#{HOMEBREW_PREFIX}/include` but instead the specific directories for specific dependencies.